.gitattributes

```diff
@@ -0,0 +1,10 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
```

Comment:
TIL about git attributes and github linguist https://medium.com/@bolajiayodeji/introducing-github-linguist-8f09273ddea1